dataactbroker/handlers/fileHandler.py

```diff
@@ -1,11 +1,14 @@
 import os
+import boto
```

Comment:
We should test one more time just to make sure there aren't any issues, especially when switching versions since we don't know exactly what is different